test/core/proofs/proofs_test.cpp

```diff
@@ -191,7 +191,7 @@ TEST_F(ProofsTest, DISABLED_ValidPoSt) {
  * @when Commit and Verifies Seal and Unseal
  * @then success
  */
-TEST_F(ProofsTest, ValidSealAndUnseal) {
+TEST_F(ProofsTest, DISABLED_ValidSealAndUnseal) {
```

Comment:
Not clear why the test gets disabled.

Please leave a todo with Jira task id.
The comment should explain why it is disabled and when it should be enabled.